tools/src/test/python/dlpx/virtualization/_internal/conftest.py

```diff
@@ -226,6 +226,116 @@ def plugin_config_content(plugin_id, plugin_name, external_version, language,
     return config
 
 
+@pytest.fixture
+def plugin_config_content_missing_minimum_lua_version(
+        plugin_id, plugin_name, external_version, language, host_types,
+        plugin_type, entry_point, src_dir, schema_file, manual_discovery,
+        build_number, lua_name):
+    """
+    This fixture is missing a minimumLuaVersion while luaName is defined.
+    """
+    config = {
+        'defaultLocale': 'en-us',
+        'rootSquashEnabled': True,
+    }
+
+    if plugin_id:
+        config['id'] = plugin_id
+
+    if plugin_name:
+        config['name'] = plugin_name
+
+    if external_version:
+        config['externalVersion'] = external_version
+
+    if language:
+        config['language'] = language
+
+    if host_types:
+        config['hostTypes'] = host_types
+
+    if plugin_type:
+        config['pluginType'] = plugin_type
+
+    if entry_point:
+        config['entryPoint'] = entry_point
+
+    if src_dir:
+        config['srcDir'] = src_dir
+
+    if schema_file:
+        config['schemaFile'] = schema_file
+
+    # Here we do an 'is not None' check because we will be passing in
+    # booleans as a parameter in tests.
+    if manual_discovery is not None:
+        config['manualDiscovery'] = manual_discovery
+
+    if build_number:
+        config['buildNumber'] = build_number
+
+    if lua_name:
+        config['luaName'] = lua_name
+
+    return config
```

Comment:
instead of having a completely new fixture can we just use the plugin_config_content and overwrite the minimum_lua_version using a ? something like this should work:
```
    @pytest.mark.parametrize('minimum_lua_version', [None])
    def test_build_lua_name_without_minimum_lua_version(plugin_config_content, src_dir,
            schema_content):
```

You will need to update to have plugin_config_content take in this variable similarly.

Reply:
Agreed.